.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,38 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
The `.idea` folder should not be commited as it is a folder created by your local webstorm that holds all of its internal options for this project.